### ISSUE TITLE
Media recognition on Windows 10

### ISSUE THREAD

**Issue by Farfie** [erengy/taiga#74]
I realize Windows 10 is very low priority, or rather it should be, let's blame GitHub for not letting me select a priority (unless I just can't see it).

I'll at least keep this simple for you: on build 9926 Windows 10, Taiga runs!
From what I attempt to use though, media recognition is not working with at least MPC-HC. I haven't tested any other programs as I don't use them.

**Comment by vendethiel** [comment]
> I realize Windows 10 is very low priority, or rather it should be, let's blame GitHub for not letting me select a priority (unless I just can't see it).

there's no priority system on github

**Comment by Farfie** [comment]
Seems that way. But there should be.

**Comment by erengy** [comment]
Hey @Farfie, thanks for the heads up. I'm actually surprised to hear that Taiga's having an issue with MPC-HC, as it simply reads the player's window title to detect the filename. For VLC and the like, Taiga does some complicated stuff, which I'd expect to be more problematic when a new version of the OS comes out. Anyway, I'll give the Technical Preview a try and see what's up.

**Comment by Farfie** [comment]
I would imagine then that the problem lies with the programs ability to do get the title name then, rather than something else first. But this is of course great to hear, as I don't think I'm going to stop using Win10 until the one after that comes out.

Unrelated, but I also wanted to express how happy I was to come here and also see what you were aiming at with 1.2. As someone who always wanted MAL updater to shape up, finding out about this program was simply awesome. And if I found any problem with it, it was the problems it was having detecting some of my older stuff.

Nothing but great news from all of this :D, keep up the amazing work!

**Comment by Farfie** [comment]
So I have some probably good news. I formatted last night, and after reinstalling (same Windows 10)... recognition suddenly works. Something tells me if I would have just reinstalled Taiga at some point after some of the update pushes they did, it would have started working in a similar fashion.
But as far as I'm concerned, this bug was resolved OS side.

Thanks for the program, and looking forward to future releases!
